### DISCUSSION TITLE
Expired certificate sudo apt-get update

### DISCUSSION THREAD

**Discussion by interstellartaurus** [linux-surface/linux-surface#599] [General]
Not really a big issue as I hadn’t run updates for a few weeks. If you get the message that surface Linux has no release file or their certificate expired, run “sudo apt-get install ca-certificates” to fix. Hope this is helpful to some. 

This is for Debian releases.

**Comment by qzed** [comment]
That's probably related to the [Let's Encrypt Root CA X3 Expiration](https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/). Updating your local trust/certificate store should fix that (as indicated by your comment). This probably affects all distributions equally.

**Reply by interstellartaurus** [reply]
Actually wasn’t sure if this was the issue or not when I booted up this morning. No one else had the issue and I figured that meant your team addressed it in an update I may have missed.

**Reply by qzed** [reply]
I don't think we had to do anything, as far as I can tell the certificate of the repo should be signed by both old and new CA. On the other hand I'm also not sure why exactly this issue happened. I thought that the new certificate should have been in the trust for a while now (as indicated by the notice I've linked) but I was probably wrong on that. I know others who had similar problems on their personal websites with this, but updating the ca-certificates/trust fixed that for them too. So I guess the new root certificate must have been added to that trust just recently.

**Comment by interstellartaurus** [comment]
Never 2 late to learn new things 😁

**Comment by interstellartaurus** [comment]
Got it!
